src/Bynder/Api/Impl/AbstractRequestHandler.php

```diff
@@ -1,26 +1,33 @@
 <?php
+
 namespace Bynder\Api\Impl;
 
 abstract class AbstractRequestHandler
 {
     protected $configuration;
 
-    public function sendRequestAsync($requestMethod, $uri, $options = [])
+    public function formRequest($uri, $requestMethod)
     {
-        $uri = sprintf(
-            'https://%s/%s', $this->configuration->getBynderDomain(), $uri
-        );
-
         if (!in_array($requestMethod, ['GET', 'POST', 'DELETE'])) {
             throw new Exception('Invalid request method provided');
         }
 
-        $request = $this->sendAuthenticatedRequest($requestMethod, $uri, $options);
+        return sprintf(
+            'https://%s/%s',
+            $this->configuration->getBynderDomain(),
+            $uri
+        );
```

Comment:
We could also call this method `public function sendRequestAsync($requestMethod, $uri, $options = [])` and move the stuff from that over here:

```suggestion
        $composedUri = sprintf(
            'https://%s/%s',
            $this->configuration->getBynderDomain(),
            $uri
        );
        return $this->sendAuthenticatedRequest($requestMethod, $composedUri, $options);
```

(combined with my suggested change below)

Reply:
I will prune this as I wanted to use it to get the header for the correlation-id (I had used the getHeader() method instead). But with your comments from the python-sdk I decided to remove the returning of fileId and correlationId so this won't be required at all.